### DISCUSSION TITLE
GitHub Desktop keeps signing me out

### DISCUSSION THREAD

**Discussion by luiginicastrocgs** [community/community#121711] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Whenever I sign into my GitHub account, all works as expected but when I close GitHub I keep getting signed out. I've tried some advice from older forums but nothing seems to work.

**Comment by dane** [comment]
This sounds like something to do with your browser and the GitHub session (cookie in your browser). I have a few follow up questions:
- When you close GitHub, are you closing a tab or the entire browser?
- What browser are you using and are you using private windows?
- Do you have plugins installed in your browser?

**Reply by luiginicastrocgs** [reply]
Sorry I should have clarified in the text, this is on my GitHub Desktop application, does that change your questions?
- I don't keep the browser open when using the GUI
- I use Firefox not on private windows
- Plugins are ad block and a couple others but it was working fine last week. I did a reset on my PC and wonder if something went wrong?

**Reply by dane** [reply]
Hey @luiginicastrocgs, I was assuming you were using the browser, my apologies. [This issue comment](https://github.com/desktop/desktop/issues/6533#issuecomment-451209059) is a few years old, but it looks like a thread to pull on. The bits that stand out to me are:
- Credentials are stored in Keychain on macOS and Credential Manager on Windows
- GitHub Desktop uses the AskPass protocol that Git also supports
- GitHub Desktop can avoid overwriting any saved credentials you might already have in your credential manager

I don't use GitHub Desktop, but these would be my next steps:
- [Follow their authentication troubleshooting guide](https://docs.github.com/en/desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github-in-github-desktop#troubleshooting-authentication-issues) to see if the log explorer reveals any clues
- Search for "GitHub" in your credential manager (Keychain or Credential Manager), specifically looking for entries for GitHub Desktop, delete them, log back in, and see if you stay logged in

Give those :point_up: a try in any order and let me know if the logs hint at anything.

**Comment by YassineCherkaoui** [comment]
Hey @luiginicastrocgs ,
Experiencing frequent sign-out issues on GitHub can be frustrating. Here are some troubleshooting steps you can try to resolve this problem:
Clear Cookies and Cache / Check Browser Settings / Disable Browser Extensions / Try a Different Browser

and if it didn't work try to Contact GitHub Support they will help you

And remember, if you need any further assistance, I'm here to help you out!
